### PR TITLE
changed html title

### PR DESCRIPTION
**Pull request by barish90**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Haikus Highway</title>
```
Comment (CDCaldwell) [inline public/index.html:27]: 🐱‍👤🐱‍👤🐱‍👤🐱‍👤🐱‍👤

**Review by CDCaldwell — COMMENTED** [review]
🙌🙌🐱‍🐉

File: public/index.html (hunk 1 above)
Comment (syedahmaham) [reply public/index.html:27]: 👏

Comment (cdixon83) [reply public/index.html:27]: Looks good!

**Review by syedahmaham — APPROVED** [review]

**Review by CDCaldwell — APPROVED** [review]

**Review by cdixon83 — APPROVED** [review]
Looks good!

**Pull request merged**